### PR TITLE
[86c05hezf][data-table] Feature/data table a11y multilevel in head

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
I've fixed keyboard (SR) navigation by table if it has multilevel header

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Review by j-mnizhek — APPROVED** [review]
Seems that works good 👍

I fixed only content for Borders example.

File: crowdin.yml [hunk 1]
```diff
@@ -29,109 +29,7 @@
...
-    "source": "/semcore/skeleton/src/translations/en.json",
-    "translation": "/%original_path%/%two_letters_code%.json",
-  },
-  {
-    "source": "/semcore/spin/src/translations/en.json",
-    "translation": "/%original_path%/%two_letters_code%.json",
-  },
-  {
-    "source": "/semcore/tag/src/translations/en.json",
+    "source": "semcore/data-table/src/translations/en.json",
```
Comment (msereniti) [inline crowdin.yml:32]: I guess it should be reverted?

File: semcore/data-table/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.44.0] - 2024-09-09
+
+### Added
+
+- Keyboard control.
```
Comment (msereniti) [inline semcore/data-table/CHANGELOG.md:9]: Keyboard navigation?

**Review by msereniti — APPROVED** [review]
Two nit comments + download translations before mergning please

File: crowdin.yml (hunk 1 above)
Comment (ilyabrower) [reply crowdin.yml]: yep

**Pull request merged**